test/jest_integration/saml_auth.test.ts

```diff
@@ -266,7 +266,7 @@ describe('start OpenSearch Dashboards server', () => {
   });
 
   it('Login to Dashboard with Hash', async () => {
-    const urlWithHash = `http://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d?_g=(filters:!(),refreshInterval:(pause:!f,value:900000),time:(from:now-24h,to:now))&_a=(description:'Analyze%20mock%20flight%20data%20for%20OpenSearch-Air,%20Logstash%20Airways,%20OpenSearch%20Dashboards%20Airlines%20and%20BeatsWest',filters:!(),fullScreenMode:!f,options:(hidePanelTitles:!f,useMargins:!t),query:(language:kuery,query:''),timeRestore:!t,title:'%5BFlights%5D%20Global%20Flight%20Dashboard',viewMode:view)`;
+    const urlWithHash = `http://localhost:5601/app/management/opensearch-dashboards/indexPatterns`;
```

Comment:
This test is about using the `#`, so is this scenario flaky?

Reply:
I think it is the page itself, cuz the original url was trying to load the entire visualization page thru selenium web driver. I just added a detail version of explanation down below.

Reply:
If the sample flights data visualization page is causing issues can we change the url to have a hash that points to another location that isn't problematic?

Reply:
Thats a good idea. I have updated the case with the get-started page in the security plugin panel - `http://localhost:5601/app/security-dashboards-plugin#/getstarted`